src/components/utils/onfidoApi.js

```diff
@@ -0,0 +1,58 @@
+import { performHttpReq } from '../utils/http'
+import Tracker from '../../Tracker'
+import forEach from 'object-loops/for-each'
+const onfidoUrl = process.env.ONFIDO_URL
+
+const errorType = (key, val) => {
+  if (key === 'document_detection') return 'INVALID_CAPTURE'
+  if (key === 'file') return 'INVALID_TYPE' // This error is only hit on corrupted PDF submission
+  if (key === 'face_detection') {
+    return val.indexOf('Multiple faces') === -1 ? 'NO_FACE_ERROR' : 'MULTIPLE_FACES_ERROR'
+  }
+}
+
+const identifyValidationError = (error) => {
+  const fields = error.fields
+  for (const key of Object.keys(fields)) {
+    const val = fields[key]
+    error = errorType(key, val[0])
+  }
+  return error
+}
+
+const serverError = ({status, response}) => {
+  Tracker.sendError(`${status} - ${response}`)
+  return 'SERVER_ERROR'
+}
+
+const handleApiError = (request, callback) => {
+  const response = JSON.parse(request.response)
+  const error = request.status === 422 ?
+    identifyValidationError(response.error) :
+    serverError(request)
+  callback(error)
+}
+
+export const postToOnfido = ({blob, documentType, side}, captureType, token, allowApiAdvancedValidation, onSuccess, onError) => {
+  if (captureType === 'face') return sendFile({blob}, 'live_photos', token, onSuccess, (request) => handleApiError(request, onError))
+  sendFile({blob, type: documentType, side, advanced_validation: allowApiAdvancedValidation}, 'documents', token, onSuccess, (request) => handleApiError(request, onError))
+}
+
+const objectToFormData = (object) => {
+  const formData = new FormData()
+  forEach(object, (value, key) => formData.append(key, value))
+  return formData
+}
+
+const sendFile = ({blob, ...extraOptions}, path, token, onSuccess, onError) => {
```

Comment:
Made it consistent by removing reference to Onfido in the other functions apart from the function that gets exported